### PR TITLE
Display more info on missing instruction

### PR DESCRIPTION
**Pull request by AlexandreSi**
Implements suggestion for https://github.com/4ian/GDevelop/discussions/4374

image

### CODE REVIEW

File: newIDE/app/src/EventsSheet/EventsTree/Instruction.js [hunk 1]
```diff
@@ -128,6 +130,41 @@ const formatValue = ({
...
+const InstructionMissing = (props: {|
+  instructionType: string,
+  isCondition: boolean,
+|}) => {
+  const { hasExtensionNamed } = React.useContext(ExtensionStoreContext);
+  const { name, behaviorName, extensionName } = getFunctionNameFromType(
+    props.instructionType
+  );
+  const extensionStoreMention = hasExtensionNamed(extensionName)
+    ? ' Try installing it from the extension store.'
```
Comment (ClementPasteau) [inline newIDE/app/src/EventsSheet/EventsTree/Instruction.js:142]: is the space before the "try" on purpose?
We should probably translate this too! so maybe remove the space here and add it before the mention when you use the mention below.
And use `<Trans>` ?

File: newIDE/app/src/EventsSheet/EventsTree/Instruction.js [hunk 2]
```diff
@@ -128,6 +130,41 @@ const formatValue = ({
...
+    : '';
+  if (behaviorName) {
+    return (
+      <span className="instruction-missing">
+        <span className="function-name">{name}</span>{' '}
+        {props.isCondition ? 'condition' : 'action'} on behavior{' '}
+        <span className="behavior-name">{behaviorName}</span> from{' '}
+        <span className="extension-name">{extensionName}</span> extension is
+        missing.
```
Comment (ClementPasteau) [inline newIDE/app/src/EventsSheet/EventsTree/Instruction.js:151]: Is there a way we can translate those parts easily?
Maybe having a `<Trans>{functionName} {type} on behavior {behavior} from {extension} extension is missing.</Trans>` ? Where each param is a span

File: newIDE/app/src/EventsSheet/EventsTree/Instruction.js [hunk 3]
```diff
@@ -187,6 +224,15 @@ const Instruction = (props: Props) => {
             parameterIndex >= 0 && parameterIndex < parametersCount;
 
           if (!isParameter) {
+            if (value === 'Unknown or unsupported instruction') {
```
Comment (ClementPasteau) [inline newIDE/app/src/EventsSheet/EventsTree/Instruction.js:227]: this feels like a magic string, should we have a Constant somewhere we can import and re-use here? like `UNKNOWN_INSTRUCTION_VALUE`

**Review by ClementPasteau — APPROVED** [review]
lgtm overall, just tiny details about translations if possible

File: newIDE/app/src/EventsSheet/EventsTree/Instruction.js (hunk 3 above)
Comment (AlexandreSi) [reply newIDE/app/src/EventsSheet/EventsTree/Instruction.js:227]: It's defined in the C++ part so I'm not sure it's easily doable

Comment (ClementPasteau) [reply newIDE/app/src/EventsSheet/EventsTree/Instruction.js:255]: oh right, then probably not.

File: newIDE/app/src/EventsSheet/EventsTree/Instruction.js [hunk 4]
```diff
@@ -138,30 +138,58 @@ const InstructionMissing = (props: {|
...
-        <span className="extension-name">{extensionName}</span> extension is
-        missing.
-        {extensionStoreMention}
-      </span>
-    );
+    if (props.isCondition) {
+      return (
+        <span className="instruction-missing">
+          <Trans>
+            {functionNode} condition on behavior {behaviorNode} from{' '}
```
Comment (ClementPasteau) [inline newIDE/app/src/EventsSheet/EventsTree/Instruction.js:156]: ❤️

**Review by ClementPasteau — APPROVED** [review]

**Pull request merged**